### PR TITLE
[ADHOC] feat: add OR_EQUAL filters to event action

### PR DESCRIPTION
**Pull request by Quazia**
🚨 Please review the [guidelines for contributing](https://github.com/boostxyz/boost-protocol/blob/main/.github/CONTRIBUTING.md) to this repository.

### Description
Adds `GREATER_THAN_OR_EQUAL` and `LESS_THAN_OR_EQUAL` to `EventAction.sol` and `EventAction.ts`

This will require a redeployment

💔 Thank you!

### CODE REVIEW

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  By using the `[ADHOC]` title prefix, you are bypassing best practice protections.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Are you sure you want to be submitting a change without including a changeset? If you're just changing docs or tests, you probably don't need to. See [the publishing section of the README](https://github.com/boostxyz/boost-protocol?tab=readme-ov-file#changesets--publishing) for more info.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c7c546a542739058a7504e49d99ddba7960e0a29
</p>

**Review by sammccord — APPROVED** [review]

File: packages/sdk/src/Actions/EventAction.ts [hunk 1]
```diff
@@ -913,6 +924,15 @@ export class EventAction extends DeployableTarget<
           criteria,
           fieldValue,
         });
+      case FilterType.LESS_THAN:
```
Comment (topocount) [inline packages/sdk/src/Actions/EventAction.ts:927]:
```suggestion
      case FilterType.LESS_THAN_OR_EQUAL:
```

**Review by topocount — COMMENTED** [review]
LG, just

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  By using the `[ADHOC]` title prefix, you are bypassing best practice protections.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Are you sure you want to be submitting a change without including a changeset? If you're just changing docs or tests, you probably don't need to. See [the publishing section of the README](https://github.com/boostxyz/boost-protocol?tab=readme-ov-file#changesets--publishing) for more info.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9f28abf72d0fb2554ca94f486aace68ae491276d
</p>

**Comment by sammccord** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  By using the `[ADHOC]` title prefix, you are bypassing best practice protections.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Are you sure you want to be submitting a change without including a changeset? If you're just changing docs or tests, you probably don't need to. See [the publishing section of the README](https://github.com/boostxyz/boost-protocol?tab=readme-ov-file#changesets--publishing) for more info.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9f28abf72d0fb2554ca94f486aace68ae491276d
</p>

**Pull request merged**